src/app/components/text-expand/index.html

```diff
@@ -1,5 +1,7 @@
 <sky-demo-page
-  pageTitle="Text expand">
+  pageTitle="Text expand"
+  [skyuxModule]="['SkyTextExpandModule', 'SkyTextExpandRepeaterModule']"
```

Comment:
Should we add a comma or "and" between modules when we have more than one?

Reply:
I figured since they're stylized with the `<stache-code>` element and separated by a space that we didn't need any kind of grammatical conjunction between them.